src/sprite.js

```diff
@@ -145,8 +145,8 @@
       // image sprite
       if (temp = properties.image) {
         this.image = temp;
-        this.width = temp.width;
-        this.height = temp.height;
+        this.width = (properties.width !== undefined) ? properties.width : temp.width;
```

Comment:
I'm not sure I'd expect a width or height of `0` to be passed in (would draw nothing), so this could probably be `this.width || temp.width` like in the animation sprite logic.

Reply:
I do think passing a width or height of 0 would be a valid thing. Imagine particles that grow from nothing. Like I'm doing :)